anime_downloader/extractors/kwik.py

```diff
@@ -15,26 +15,20 @@ class Kwik(BaseExtractor):
 
     def _get_data(self):
 
-        # Need a javascript deobsufication api/python, so someone smarter
-        # than me can work on that for now I will add the pattern I observed
-
-        # alternatively you can pattern match on `src` for stream_url part
-        source_parts_re = re.compile(r'action=\"([^"]+)\".*value=\"([^"]+)\".*Click Here to Download',
-                                     re.DOTALL)
-
         # Kwik servers don't have direct link access you need to be referred
         # from somewhere, I will just use the url itself.
 
         download_url = self.url.replace('kwik.cx/e/', 'kwik.cx/f/')
```

Comment:
nit: download_url -> site_url.
download_url = site_url.replace('kwik.cx/f/','kwik.cx/d/')

no need to inline so much.